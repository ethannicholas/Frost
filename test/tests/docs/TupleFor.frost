method main() {
    for (i, v) in ["Zero", "One", "Two", "Three"].enumeration {
        Console.printLine("\{i}: \{v}")
    }
}
