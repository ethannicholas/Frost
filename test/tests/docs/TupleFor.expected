0: Zero
1: One
2: Two
3: Three
